.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
```

Comment:
I don't want to switch to Travis. My idea is to use CircleCI with **docker** and TravisCI without docker, like prometheus does. Is it ok for you?